### ISSUE TITLE
Unable to add mega.nz links

### ISSUE THREAD

**Issue by 8bitgentleman** [seansfkelley/nas-download-manager#193]
### Description

I am unable to add mega.nz links via synology download manager extension. Trying to add the link either manually or via right click fails after a long wait due to an "Unknown Error". I am able to add the same url just fine via the official Download Station UI and via the other (mostly inferior) chrome synology extension "Synology Download Station".

### Steps to Reproduce the Issue

1. Try to add a [test mega.nz link](https://mega.nz/file/0wIUAIbT#r2mYt3WBdbdTa6XCTL9v0SBI8K9K3NufwxJUlhM37Sw)

### Failure Logs

```
Redacted extension state: {
  "lastSevereError": "(omitted for brevity)",
  "settings": {
    "badgeDisplayType": "total",
    "connection": {
      "hostname": true,
      "password": true,
      "port": true,
      "protocol": "http",
      "username": true
    },
    "notifications": {
      "completionPollingInterval": 60,
      "enableCompletionNotifications": true,
      "enableFeedbackNotifications": true
    },
    "shouldHandleDownloadLinks": true,
    "taskSortType": "timestamp-added-asc",
    "visibleTasks": {
      "completed": true,
      "downloading": true,
      "errored": true,
      "other": true,
      "uploading": true
    }
  },
  "stateVersion": 5,
  "taskFetchFailureReason": null,
  "tasks": 1,
  "tasksLastCompletedFetchTimestamp": 1622808576487,
  "tasksLastInitiatedFetchTimestamp": 1622808576298
}

3 error(s) while resolving URLs; first message: error while trying to fetch metadata file type for download url

TypeError: 'Failed to fetch'
Error stack trace: TypeError: Failed to fetch
```

### Versions

**Extension version**: 0.10.3

**Browser version**: Chromium: 91.0.4472.77 (Official Build) (x86_64)

**DSM version**: 6.2.4

**Comment by seansfkelley** [comment]
Hey there, thanks for the report!

I took a look at this, and while there was indeed a bug with the error reporting (since fixed in v0.11.0), it appears that [the way that site works isn't compatible with this extension](https://github.com/seansfkelley/nas-download-manager/blob/master/FAQ.md#why-cant-i-start-a-download-from-a-site-or-how-do-i-start-a-download-with-a-site). To get technical, instead of providing a link you can send to the NAS, it downloads the file onto the page, decrypts it, and then programmatically triggers a download from a blob.

**Comment by 8bitgentleman** [comment]
Thanks for looking into this! If that's the case how is it that Download Station handles these links? Seems like it uses a [pyload plugin](https://github.com/pyload/pyload/blob/07e5e009d177b7ed974d2e0ecd61c3ea262af21b/src/pyload/plugins/accounts/MegaCoNz.py) for it and gets around the blob issue by using the mega api. Is that something nas-download manager could do as well? (side note did the name of this plugin change today or am I going crazy?) 
![image](https://user-images.githubusercontent.com/4028391/120872799-b3962200-c56d-11eb-8fd1-2ed5e6e658b5.png)

**Comment by seansfkelley** [comment]
I see, in that case, that seems like something I should support. Thanks for providing that example!

**Comment by 8bitgentleman** [comment]
Awesome, thanks for reopening!

**Comment by seansfkelley** [comment]
Alright, this should be fixed in 0.11.1. Chrome should approve the new release within a day or two.
